pkg/kube/wait.go

```diff
@@ -223,26 +222,6 @@ func (hw *legacyWaiter) WatchUntilReady(resources ResourceList, timeout time.Dur
 	return perform(resources, hw.watchTimeout(timeout))
 }
 
-func perform(infos ResourceList, fn func(*resource.Info) error) error {
```

Comment:
this was simply moved to `client.go` (https://github.com/helm/helm/pull/31030/files#diff-137bba79988de442c1fde0fc29520d6d7e5a5239ceac82f5439a5bf513ebb2d4R817). Not sure why it was in `wait.go`